npm-update.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
likewise - superfluous

Reply:
it is for my JetBrains environment to check shell cmds in files via shellcheck